web-components/src/components/phone-input/PhoneInput.ts

```diff
@@ -152,6 +152,13 @@ export namespace PhoneInput {
       this.validateInput(this.value);
       this.validateNumber();
       event.stopPropagation();
+      // Strip when Phone number starts with selected Country Code.
+      const countryCodeValue = this.countryCallingCode.split(",")[0]?.trim();
```

Comment:
extract a separate function for all these logics. 
is there no way to handle this in desktop code?

Reply:
Logic is changed to separate function, yes it's possible to handle this in desktop code but vivek recommendation was do it in momentum.